src/Grid.cpp

```diff
@@ -0,0 +1,114 @@
+#include "Grid.hpp"
+#include <fstream>
+#include "common.hpp"
+
+namespace mathboard 
+{
+  Grid::Grid(const cv::Point2f &topLeftCorner, const cv::Point2f &botRightCorner, float cellSize) :
+      m_TopLeftCorner(topLeftCorner), m_BotRightCorner(botRightCorner),
+      m_CellSize(cellSize),
+      m_Height((m_BotRightCorner.y - m_TopLeftCorner.y) / m_CellSize),
+      m_Width((m_BotRightCorner.x - m_TopLeftCorner.x) / m_CellSize) 
+  {
+    if(m_CellSize <= 0) 
+    {
+        std::ofstream debug_stream("debug_output.txt", std::ios::app);
+        debug_stream << "Cell size have to be positive number\n";
+        debug_stream.close();
+    }
+    m_Grid.resize(m_Height);
```

Comment:
Add some white space or use a formatter that will automatically do that for you. I use LLVM formatting style so let's just stick to that.